lib/alephant/broker/response/base.rb

```diff
@@ -35,7 +35,7 @@ def setup; end
         private
 
         def log_status
-          add_no_cache_headers if status !~ /200/
+          add_no_cache_headers if status != 200
         end
 
```

Comment:
To be honest (although I like one line methods) I think this line (unless used in multiple places) should be inlined from where it's called; as it's just an empty indirection around some code already in directed via a `add_no_cache_headers`